### ISSUE TITLE
[WebToolsE2E] Aspire.ProjectTemplates should be marked as Reserved

### ISSUE THREAD

**Issue by balachir** [dotnet/aspire#6485]
**INSTALL STEPS**
1. Clean Windows x64 machine
2. Install VS 17.12 GA
3. Install Aspire.ProjectTemplates version 9.0.0

**REPRO STEPS**
1. Open command prompt, and run `dotnet new uninstall`

**ACTUAL**
Aspire.ProjectTemplates is listed, and it is marked as 'Not Reserved'

![Image](https://github.com/user-attachments/assets/d600de64-a73f-453b-84fa-3f11bd491d2d)

**EXPECTED**
Aspire.ProjectTemplates should be marked as 'Reserved' (i.e. Reserved with a check mark next to it)

**Comment by DamianEdwards** [comment]
I don't know what this "Reserved" marker is. Are there any more details? @phenning @sayedihashimi @baronfel

**Comment by baronfel** [comment]
I believe that signaling that the package name has reserved ownership on nuget.org - it's one of their trust signaling mechanisms I believe.

**Comment by sayedihashimi** [comment]
I'm not familiar with that, @baronfel  should have more details.

**Comment by DamianEdwards** [comment]
> I believe that signaling that the package name has reserved ownership on nuget.org

Hmm, it does, but it's reserved by the "aspire" nuget.org account.

![Image](https://github.com/user-attachments/assets/da18d15f-b459-45cb-99a5-21b4a26a95e1)

**Comment by DamianEdwards** [comment]
What does `dotnet new` to determine the status of the reserved flag it's showing?

**Comment by baronfel** [comment]
I checked the code and we're just flowing through a Boolean 'verified' flag that's visible from the NuGet metadata query APIs directly.

**Comment by DamianEdwards** [comment]
That makes sense then as these packages aren't on nuget.org yet. This will only ever show verified when installed from nuget.org

**Comment by DamianEdwards** [comment]
@balachir I think this is expected until the packages are on nuget.org. Have you seen it say reserved for packages installed from AzDO feeds previously?

**Comment by balachir** [comment]
@DamianEdwards @baronfel thanks for investigating this. No, actually I haven't seen it say reserved for packages from internal feeds before. I just saw this flag recently with our Aspire 9.0-rc1 templates when I installed from nuget.org. I was just wondering if this is something that needs to set in the templates for Aspire 9.0 GA before we ship. But it sounds like we are good here. I'll go ahead and close this issue.
